autonomy/analyse/abci/docstrings.py

```diff
@@ -78,41 +76,26 @@ def docstring_abci_app(abci_app: Any) -> str:  # pylint: disable-msg=too-many-lo
     )
 
 
-def update_docstrings(
-    module_path: Path, docstring: str, abci_app_name: str, check: bool = False
-) -> bool:
```

Comment:
This function was doing so many things at a time, so I refactored into multiple functions which serves one purpose